server/index.js

```diff
@@ -427,6 +449,7 @@ app.get('/api/videodevices', (req, res) => {
         error: null,
         fps: fps,
         FPSMax: FPSMax,
+        useCameraHeartbeat,
```

Comment:
enableCameraHeartbeat: useCameraHeartbeat,

Reply:
Well that is an embarrassing thing to miss! Thanks for finding it.